test/network_interface_test.cpp

```diff
@@ -5,50 +5,51 @@
 * See file LICENSE included with this software or go to https://opensource.org/licenses/MIT for full license details.
 */
 
-#include <network_utils.h>
+#include <network_interface/network_utils.h>
 #include <gtest/gtest.h>
-#include <stdio.h>
+
+#include <vector>
 
 TEST(TCPInterface, testReadBEFloat)
 {
   // 123.4567 as binary float
-  uint8_t bytes[4] = { 0x42, 0xF6, 0xE9, 0xD5};
-  ASSERT_EQ(123.45670318603515625, AS::Network::read_be<float>(bytes, 4, 0));
+  std::vector<uint8_t> bytes = { 0x42, 0xF6, 0xE9, 0xD5};
```

Comment:
Doesn't matter at all, but the brace spacing is inconsistent in all these initializers